Modules/_testcapi/parts.h

```diff
@@ -1,9 +1,36 @@
-#include "Python.h"
+#ifndef Py_TESTCAPI_PARTS_H
+#define Py_TESTCAPI_PARTS_H
+
+#include "pyconfig.h"  // for Py_TRACE_REFS
+
+// Figure out if Limited API is available for this build. If it isn't we won't
+// build tests for it.
+// Currently, only Py_TRACE_REFS disables Limited API.
+#ifdef Py_TRACE_REFS
+#undef LIMITED_API_AVAILABLE
+#else
+#define LIMITED_API_AVAILABLE 1
```

Comment:
<details>
<summary> ⛏️ ⚠️ </summary>

Just for fun: you _could_ `#define LIMITED_API_AVAILABLE 0` instead of `#undef`fing it and then simply use `PyModule_AddIntMacro` later (iso. `PyModule_AddObjectRef`.

The current implementation is of course perfectly fine, so you can disregard this comment :) 

</details>

Reply:
I'd like to check this with `#ifdef`, like all the other feature macros (see `pyconfig.h`)